packages/browser-sdk/src/feature/features.ts

```diff
@@ -291,11 +324,41 @@ export class FeaturesClient {
     return checkEvent.value;
   }
 
-  private setFeatures(features: RawFeatures) {
-    this.features = features;
+  private triggerFeaturesChanged() {
+    const mergedFeatures: RawFeatures = {};
+
+    // merge fetched features with overrides into `this.features`
+    for (const key in this.fetchedFeatures) {
+      const fetchedFeature = this.fetchedFeatures[key];
+      if (!fetchedFeature) continue;
+      const isEnabledOverride = this.featureOverrides[key] ?? null;
+      mergedFeatures[key] = {
+        ...fetchedFeature,
+        isEnabledOverride,
+      };
+    }
+
+    // add any overrides that aren't in the fetched features
+    for (const key in this.featureOverrides) {
+      if (!this.features[key]) {
+        mergedFeatures[key] = {
+          key,
+          isEnabled: false,
+          isEnabledOverride: null,
+        };
+      }
+    }
```

Comment:
why is this useful? especially when `isEnabled` and `isEnabledOverride` is always set to `false` and `null` instead of the actual overriden value?

Reply:
> isEnabledOverride is always set to false and null instead of the actual overriden value?

that's a bug. Thanks!

fetched features is what the API returns. If you've added a feature locally but not on app.bucket.co we should still include it.

Reply:
right 👍